force_gromacs/commands/gromacs_commands.py

```diff
@@ -161,7 +191,3 @@ class Gromacs_trjconv(BaseGromacsCommand):
                       '-trunc', '-exec', '-split', '-sep', '-nosep',
                       '-nzero', '-dropunder', '-dropover', '-conect',
```

Comment:
`'-conect'` or `'-connect'`?

Reply:
I double checked, correct flag is `-conect` (refers to `conect` records in Gromacs output)

Reply:
Thank you, sorry for bothering you.